client/config.yaml

```diff
@@ -29,5 +29,12 @@ translation_config:
   translation_type: Translation_Teradata2BQ
   location: 'us'
 
+  # The default database name to fully qualify SQL objects when their database name is missing.
```

Comment:
Maybe highlight this as an optional field?  What about putting [Optional field] here?